### ISSUE TITLE
TaskService: Saved Sessions (Persistent Filter)

### ISSUE THREAD

**Issue by darenegade** [it-at-m/digiwf-core#288]
**Is your feature request related to a problem? Please describe.**
As a user of the task list, I need a way to save the current session (previously "saved search"):

![image.png](https://images.zenhubusercontent.com/62f38045a2e3a269d82cb8b8/00c6a2a7-4135-4050-ab4f-762ca7c0cf96)

**Describe the solution you'd like**
Store Session of specific View (User/Group/Process) in Fronted (VueX) and save to local storage (vuex plugin)

Migration Path: 
Request all Saved-Queries from Backend at startup and migrate them to local storage

**Describe alternatives you've considered**
Use Browser Bookmark -> UX Issue for some Users

**Additional context**
Storage of process/task names (tags) are handled in #542.
Additional filters and sort are now added, e.g. sort #512

**Comment by lmoesle** [comment]
**Proposal for acceptance criteria:**

- [ ] The saved searches should be stored in local storage for each user
- [ ] The saved searches only makes a single http requests during startup to migrate saved searches to the local storage
- [ ] Migrated saved searches are marked for deletion in the backend - **Assumption**
- [ ] Migrated saved searches are automatically deleted after some time (e.g. 30 days) - **Assumption**

**Questions:**

- What happens to saved searches after migration? Do we delete them (directly after migrations or in 2 steps)?

**Comment by darenegade** [comment]
> What happens to saved searches after migration? Do we delete them (directly after migrations or in 2 steps)?

Yes, one of the goals of this ticket is to reduce the complexity of the backend, as this feature is not used often and is mainly addressed with the #542 tag.
